app/driver/onewire.c

```diff
@@ -180,7 +180,6 @@ void onewire_write(uint8_t pin, uint8_t v, uint8_t power /* = 0 */) {
   if ( !power) {
```

Comment:
I think the calls to noInterrupts()/interrupts() should be removed in this PR as it's a direct consequence of the change being done, i.e. keep all related changes together. And it's one less thing Terry will have to do :)